build/eirini/_vendir/eirini/core/migration-job.yml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
What is this job doing?

Reply:
We have effectively implemented a migration framework for Eirini StatefulSets, and this job triggers it. At the moment we have one migration in place, it's needed to retroactively fix a bug in the way Eirini calculated CPU requests, which lead to very inefficient scheduling of workloads on nodes. We'd like the Job to run at every upgrade, and it's designed to only apply the migrations that are missing for any particular StatefulSet.

Reply:
@jspawar Any further concern here?

Reply:
Jwal and I reviewed this together yesterday and this answers the questions we had. Thanks @gcapizzi.

My only other ask is that that context about the migration Job be captured in the "Eirini is composed of:" of its [README](https://github.com/cloudfoundry-incubator/eirini/blob/master/README.md), but that shouldn't block this PR.

Reply:
@tcdowney @jspawar Thanks! We can move this forward.